docs/source/pythonapi/examples/mgxs-part-iii.ipynb

```diff
@@ -1108,7 +1104,7 @@
      "name": "stderr",
      "output_type": "stream",
      "text": [
-      "/home/romano/openmc/openmc/tallies.py:1506: RuntimeWarning: invalid value encountered in true_divide\n",
+      "/home/nelsonag/git/openmc/openmc/tallies.py:1506: RuntimeWarning: invalid value encountered in true_divide\n",
```

Comment:
I have, as I think you've realized in a later comment.  For benefit of others: in the Library you set the MGXS types with strings. When the Library sees 'nu-fission', it sets `nu=True` during the initialization of the `FissionXS` class.

- [x]